src/main/resources/db/migration/V20170713134558340__create_item_sequence.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER //
+CREATE PROCEDURE CreateSequence (`name` VARCHAR(30), `start` INT, `inc` INT)
+  BEGIN
+    CREATE TABLE IF NOT EXISTS `_sequences`
```

Comment:
Not sure if it really pertains but the syntax `CREATE TABLE IF NOT EXISTS` works fine outside a procedure.